mythril/analysis/modules/integer.py

```diff
@@ -156,14 +169,30 @@ def _handle_sstore(self, state):
             issue.description = "This binary {} operation can result in {}.\n".format(
                 annotation.operator, title.lower()
             )
+            address = ostate.get_current_instruction()["address"]
```

Comment:
can we make a private function for this as the same logic is being done at ln 68